dkist/dataset/utils.py

```diff
@@ -10,15 +10,26 @@
 
 
 def dataset_info_str(ds):
+    # Check for an attribute that only appears on TiledDataset
+    # Not using isinstance to avoid circular import
+    is_tiled = hasattr(ds, "combined_headers")
+    dstype = type(ds).__name__
+    if is_tiled:
+        tile_shape = ds.shape
+        ds = ds[0, 0]
     wcs = ds.wcs.low_level_wcs
 
     # Pixel dimensions table
 
-    instr = ds.meta.get("instrument_name", "")
+    instr = ds.inventory.get("instrument_name", "")
```

Comment:
If this starting to work doesn't cause a test to fail something has gone wrong.
```suggestion
    instr = ds.inventory.get("instrument", "")
```